src/core_modules/capture-core/components/WidgetErrorAndWarning/WidgetWarning/WidgetWarning.types.ts

```diff
@@ -0,0 +1,9 @@
+import type { Message } from '../content/WidgetErrorAndWarningContent.types';
+
+export type Props = {
+    warning?: Array<Message> | null;
+};
+
+export type PlainProps = Props & {
+    classes?: string;
+};
```

Comment:
Delete PlainProps and import and use Props in src/core_modules/capture-core/components/WidgetErrorAndWarning/WidgetWarning/WidgetWarning.component.tsx